src/views/Terms/index.jsx

```
@@ -1,7 +1,158 @@
+          <li>
+            You are solely responsible for any activity that occurs under your
+            account.
+          </li>
+        </ul>
+      </section>
+
+      <section>
+        <h2 className="text-2xl font-bold text-black mt-8">3. User Content</h2>
+        <p>
+          You may upload videos, images, open letters, comments, and other
+          content ("User Content") under the following conditions:
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)